modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/template/RESTClientTemplateContextContributor.java

```diff
@@ -59,6 +61,17 @@ public RESTClient(
 		}
 
 		public Object get(String path) throws Exception {
+			try {
+				return _get(path);
+			}
+			catch (Throwable throwable) {
+				_log.error(throwable, throwable);
+
+				throw throwable;
+			}
+		}
+
+		private Object _get(String path) throws Exception {
```

Comment:
What about testing it? Is it possible? I miss the tests :cry:

Reply:
We have no integration tests for RESTClient. I could not make them work when I tried.